Windows/ppsspp.rc

```diff
@@ -187,7 +187,8 @@ BEGIN
     PUSHBUTTON      "Step Out",IDC_STEPOUT,228,0,40,14
     PUSHBUTTON      "Next HLE",IDC_STEPHLE,276,0,40,14
     PUSHBUTTON      "Breakpoint",IDC_MEMCHECK,317,0,45,14
-    EDITTEXT        IDC_THREADNAME,370,3,150,10,ES_AUTOHSCROLL | ES_READONLY | NOT WS_BORDER
+    EDITTEXT        IDC_THREADNAME,365,3,150,10,ES_AUTOHSCROLL | ES_READONLY | NOT WS_BORDER

```

Comment:
Few like above there is `IDD_DISASM DIALOGEX 0, 0, 510, 415` that should have his width changed too now (at least I think, never used this kind of file and I'm not even on windows...).